modules/orientdb/src/test/java/org/testcontainers/containers/OrientDBContainerTest.java

```diff
@@ -11,7 +12,7 @@
  */
 public class OrientDBContainerTest {
 
-    private static final DockerImageName ORIENTDB_IMAGE = DockerImageName.parse("orientdb:3.0.24-tp3");
+    private static final DockerImageName ORIENTDB_IMAGE = DockerImageName.parse("orientdb:3.2.0-tp3");
```

Comment:
Assuming this PR is backward compatible, WDYT about having tests for both tags, so it can be explicit?

Reply:
it will fail because of the driver IIRC. I was wondering if we can have smoke tests for those cases or examples will be ok? 🤔

Reply:
If you checked it manually, I think it is ok for now. Adding an additional example project sounds like too much overhead to me.

Reply:
agree! given our current structure. In the future maybe make sense